integration.json

```diff
@@ -1,5 +1,6 @@
 {
   "cpython":"github.com/paketo-buildpacks/cpython",
   "pip":"github.com/paketo-buildpacks/pip",
+  "poetry-install":"github.com/paketo-buildpacks/poetry-install",
```

Comment:
I don't think we should have references to `poetry-install` in this buildpack.

We can test the behavior of this buildpack (and its dependency on `cpython`) without actually pulling in the dependencies.